CHANGELOG.md

```diff
@@ -5,6 +5,160 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
 tries to adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.2.3](https://github.com/oscar-system/Oscar.jl/releases/tag/v1.2.3) - 2025-02-27
+
+The following gives an overview of the changes compared to the previous release. This list is not
+complete, many more internal or minor changes were made, but we tried to only list those changes
+which we think might affect some users directly.
+
+### Highlights
+
+- [#4399](https://github.com/oscar-system/Oscar.jl/pull/4399) Graduate Weyl groups and root systems from experimental to officially supported 🎉  
```

Comment:
I've added a 🎉 to the PR title -- maybe too much? :-) feel free to remove it there